tests/nightly/test_large_array.py

```diff
@@ -1199,6 +1200,57 @@ def test_full():
     assert a[-1][-1] == 3
 
 
+def test_load_save():
+    x = create_2d_tensor(SMALL_Y, LARGE_X)
+    nd.save('large_tensor', [x])
+    y = nd.load('large_tensor')
+    y = y[0]
+    assert x[0][0] == y[0][0]
+    assert x[-1][-1]== y[-1][-1]
+    os.remove('large_tensor')
```

Comment:
just create a directory in tmp using mkdtemp and you dont have to worry about removing.

Reply:
done